src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -2,13 +2,16 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
-   href="${publishButton.actionLink}"
+<%@ attribute name="showButtonAndTooltip" required="true" %>
```

Comment:
The other attributes here are not dependent on `publishButton` so you don't have to add the `showButtonAndTooltip` in the FeedbackSessionPublishButton class.